app/models/concerns/Auditable.rb

```diff
@@ -0,0 +1,10 @@
+# Concern Audiable for Models
```

Comment:
Do me a solid and lowercase this filename? Should be `auditable.rb`, just for convention's sake.